setup.py

```diff
@@ -39,7 +39,7 @@ def load_requirements(filename):
 
 setup(
     name="correios",
-    version="2.0.1",
+    version="2.1.0",
```

Comment:
no need to update the version here.

in this lib version updates are generated via the `bumpversion` command. @osantana is the on who does that because he has access to the pypi. so he does all the correios lib releases